ow_lander/scripts/trajectory_publisher.py

```diff
@@ -131,24 +133,50 @@ def talker(req):
       print "Sent %s on joint[0] publisher"%(float("%14s\n"%row[12+0]))
       rate.sleep()
 
-        
+
   if guarded_move_bool : # If the activity is a guarded_move
+
+    # Start publisher for guarded_move result message
+    pubs.append(rospy.Publisher('/guarded_move_result', GuardedMoveResult, queue_size=10))
+
```

Comment:
I suggest that you move this piece to the top along with the other publishers

Reply:
I put it there because this particular publisher should exist only if the published trajectory refers to guarded_move, while the other publishers exist for every trajectory.